plutoserver/__init__.py

```diff
@@ -1,13 +1,11 @@
 import os
 
+
 def setup_plutoserver():
-  mydir = os.path.dirname(os.path.realpath(__file__))
-  iconpath = os.path.join(mydir, 'icons', 'pluto-logo.svg')
-  return {
-    "command": ["julia", "--optimize=0", "-e", "import Pluto; Pluto.run(host=\"0.0.0.0\", port={port}, launch_browser=false, require_secret_for_open_links=false, require_secret_for_access=false)"],
-    "timeout": 60,
-    "launcher_entry": {
-        "title": "Pluto.jl",
-        "icon_path": iconpath
-    },
-  }
+    mydir = os.path.dirname(os.path.realpath(__file__))
+    iconpath = os.path.join(mydir, "icons", "pluto-logo.svg")
+    return {
+        "command": ["/bin/bash", "/home/jovyan/runpluto.sh", "{port}"],
```

Comment:
I'm not sure if this should be made configurable somehow, since the /home/jovyan is specific to the docker images.

Reply:
Yes it should, you're absolutely right. Do you have an idea where to put that file to have consistent access to it?

I tried `~/runpluto.sh` but it couldn't find it

Reply:
I'm not sure what the "Python" way is to do this kind of configuration. Two things I can think of is to read parameters from a config file under the home directory, or use an environment variable to override the location.

Reply:
Well, that's an escape hatch out of python anyway, so maybe it's best to copy to /usr/bin/ and chmod +x in the Dockerfile, and just run it as an executable